### PR TITLE
WHCM fixes for spectrum-css

### PR DESCRIPTION
**Pull request by jnurthen**
This fixes Windows High Contrast mode for the following components:
* accordion
* actionbutton
* avatar
* breadcrumb
* button
* calendar
* clearbutton
* closebutton
* menu
* progressbar
* progresscircle
* radio
* rating
* sidenav
* slider
* statuslight
* stepper
* switch
* tag
* textfield
* toast
* tooltip

This is a forward port of react-spectrum PRs 
* https://github.com/adobe/react-spectrum/pull/2620 
* https://github.com/adobe/react-spectrum/pull/2626
* https://github.com/adobe/react-spectrum/pull/2830

## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [ ] This pull request is ready to merge.

### CODE REVIEW

File: components/calendar/skin.css [hunk 1]
```diff
@@ -139,3 +139,71 @@ governing permissions and limitations under the License.
...
+    &:not(.is-today) {
+      /* &:before {
+        content: none;
+      }  */
+    }
+    &.is-today {
+      /* &:before {
+        forced-color-adjust: auto;
+      } */
+    }
```
Comment (pfulton) [inline components/calendar/skin.css:182]: Are we safe to remove these lines, or do we need to keep them around for future reference?

**Review by pfulton — CHANGES_REQUESTED** [review]
This is awesome, @jnurthen. Thanks so much for taking care of these changes. I had just one question here.

File: components/calendar/skin.css (hunk 1 above)
Comment (jnurthen) [reply components/calendar/skin.css:182]: please remove - leaving was an oversight

**Review by lazd — APPROVED** [review]
LGTM, just remove any dead code!

File: components/calendar/skin.css (hunk 1 above)
Comment (pfulton) [reply components/calendar/skin.css:182]: @jnurthen ok, I just wanted to double-check. Would you mind removing these lines & pushing up that change? We can merge after that. Thanks!

**Review by pfulton — APPROVED** [review]

**Comment by GarthDB** [comment]
Released.

**Pull request merged**